### PR TITLE
Digital Shadows package creation using V2 script.

### PR DESCRIPTION
**Pull request by yashwanth-metron**
Change(s):
   - Package created using V2 script

   Reason for Change(s):
   - New Package creation script used

   Version Updated:
   - Yes

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: Solutions/Digital Shadows/Package/createUiDefinition.json [hunk 1]
```diff
@@ -60,17 +60,7 @@
             "name": "dataconnectors1-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This Solution installs the data connector for Digital Shadows SearchLight. You can get Digital Shadows SearchLight custom log data in your Microsoft Sentinel workspace. Configure and enable this data connector in the Data Connector gallery after this Solution deploys. This data connector creates custom log table(s) DigitalShadows_CL in your Microsoft Sentinel / Azure Log Analytics workspace."
```
Comment (kapil-metron) [inline Solutions/Digital Shadows/Package/createUiDefinition.json:63]: Make sure the text is same as it was in last version 1.0.0

File: Solutions/Digital Shadows/Package/createUiDefinition.json [hunk 2]
```diff
@@ -236,8 +218,7 @@
       "workspace-location": "[first(map(filter(basics('getLAWorkspace').value, (filter) => and(contains(toLower(filter.id), toLower(resourceGroup().name)),equals(filter.name,basics('workspace')))), (item) => item.location))]",
       "location": "[location()]",
       "workspace": "[basics('workspace')]",
-      "workbook1-name": "[steps('workbooks').workbook1.workbook1-name]",
-      "playbook1-PlaybookName": "[steps('playbooks').playbook1.playbook1-PlaybookName]"
+      "workbook1-name": "[steps('workbooks').workbook1.workbook1-name]"
```
Comment (kapil-metron) [inline Solutions/Digital Shadows/Package/createUiDefinition.json:221]: Playbook is not included in this file, can you check why it is missing?

**Review by kapil-metron — CHANGES_REQUESTED** [review]

File: Tools/Create-Azure-Sentinel-Solution/V2/input/Solution_DigitalShadows.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    ],
+    "Workbooks": [
+        "Workbooks/DigitalShadows.json"
+    ],
+    "Playbooks": [
+        "Playbooks/DigitalShadowsPlaybook-UpdateIncidentStatus.json"
+    ],
+    "Metadata": "SolutionMetadata.json",
+    "BasePath": "/home/vb/Desktop/Github/azure/Azure-Sentinel/Solutions/Digital Shadows",
+    "Version": "1.1.0",
```
Comment (devang-metron) [inline Tools/Create-Azure-Sentinel-Solution/V2/input/Solution_DigitalShadows.json:21]: Should it be 1.0.0?

File: Solutions/Digital Shadows/Package/mainTemplate.json [hunk 4]
```diff
@@ -674,22 +674,7 @@
...
-            },
-            {
-              "description": "**Option 2 - Manual Deployment of Azure Functions**\n\n Use the following step-by-step instructions to deploy the 'Digital Shadows SearchLight' connector manually with Azure Functions."
```
Comment (devang-metron) [inline Solutions/Digital Shadows/Package/mainTemplate.json:680]: @kapil-metron Are we removing this options?

**Review by devang-metron — CHANGES_REQUESTED** [review]
Shouldn't the package version be 1.0.0.zip?
why removed Tools/Create-Azure-Sentinel-Solution/V2/input/Solution_DigitalShadows.json?

File: Tools/Create-Azure-Sentinel-Solution/V2/input/Solution_DigitalShadows.json (hunk 3 above)
Comment (yashwanth-metron) [reply Tools/Create-Azure-Sentinel-Solution/V2/input/Solution_DigitalShadows.json:21]: 1.1.0

File: Tools/Create-Azure-Sentinel-Solution/V2/input/Solution_DigitalShadows.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+      "Analytic Rules/Digital_Shadows_incident_creation_include.yaml"
+    ],
+    "Workbooks": [
+        "Workbooks/DigitalShadows.json"
+    ],
+    "Playbooks": [
+        "Playbooks/DigitalShadowsPlaybook-UpdateIncidentStatus.json"
+    ],
+    "Metadata": "SolutionMetadata.json",
+    "BasePath": "/home/vb/Desktop/Github/azure/Azure-Sentinel/Solutions/Digital Shadows",
```
Comment (devang-metron) [inline Tools/Create-Azure-Sentinel-Solution/V2/input/Solution_DigitalShadows.json:20]: @kapil-metron Should we change the base path also?

**Review by devang-metron — CHANGES_REQUESTED** [review]

File: Solutions/Digital Shadows/Package/mainTemplate.json (hunk 4 above)
Comment (kapil-metron) [reply Solutions/Digital Shadows/Package/mainTemplate.json:680]: @devang-metron , Yes, Manual deployment would be very difficult for the user to configure, so lets remove it.

File: Solutions/Digital Shadows/Package/createUiDefinition.json [hunk 6]
```diff
@@ -114,13 +110,13 @@
...
                 "options": {
-                  "text": "For gaining insights into Digital Shadows logs."
+                  "text": "A workbook that provides a dashboard to gain insights into Digital Shadows alerts."
                 }
               },
               {
                 "name": "workbook1-name",
                 "type": "Microsoft.Common.TextBox",
-                "label": "Display Name",
+                "label": "Display name",
```
Comment (devang-metron) [inline Solutions/Digital Shadows/Package/createUiDefinition.json:119]: Keep it `Display Name`

File: Tools/Create-Azure-Sentinel-Solution/V2/input/Solution_DigitalShadows.json [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+      "Analytic Rules/Digital_Shadows_incident_creation_include.yaml"
+    ],
+    "Workbooks": [
+        "Workbooks/DigitalShadows.json"
+    ],
+    "Playbooks": [
+        "Playbooks/DigitalShadowsPlaybook-UpdateIncidentStatus.json"
+    ],
+    "Metadata": "SolutionMetadata.json",
+    "BasePath": "/home/metron/Desktop/Github/Azure-Sentinel/Solutions/Digital Shadows",
```
Comment (devang-metron) [inline Tools/Create-Azure-Sentinel-Solution/V2/input/Solution_DigitalShadows.json:20]: Make the base path changes as required.
@kapil-metron Please take a look at this.

**Review by devang-metron — CHANGES_REQUESTED** [review]

**Review by kapil-metron — APPROVED** [review]

**Pull request merged**